elements/bulbs-slick-slideshow/bulbs-slick-slideshow.js

```diff
@@ -17,7 +19,7 @@ import './bulbs-slick-slideshow.scss';
 /// 7. On l/r keydown, trigger slidechange.
 class BulbsSlickSlideshow extends BulbsHTMLElement {
   attachedCallback () {
-    this.slideshow = $('.slider');
+    this.slideshow = $('.slider').last();
```

Comment:
is `.slider` inside the `<bulbs-slick-slideshow>` element?

Could this be scoped to

```js
this.slideshow = $(this).find('.slider').last();
```